src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -114,76 +114,87 @@
 					</template>
-					<Hint v-else-if="noMatchFound"
-						:hint="t('spreed', 'No matches found')" />
-					<template v-if="isSearching">
-						<template v-if="!listedConversationsLoading && searchResultsListedConversations.length > 0">
+
+					<!-- Search results -->
+					<template v-else-if="isSearching">
+						<!-- Search results: user's conversations -->
+						<Conversation v-for="item of searchResultsConversationList"
+							:key="item.id"
+							:ref="`conversation-${item.token}`"
+							:item="item" />
+						<Hint v-if="searchResultsConversationList.length === 0" :hint="t('spreed', 'No matches')" />
+
+						<!-- Search results: listed (open) conversations -->
+						<template v-if="!listedConversationsLoading && searchResultsListedConversations.length !== 0">
 							<NcAppNavigationCaption :title="t('spreed', 'Open conversations')" />
 							<Conversation v-for="item of searchResultsListedConversations"
 								:key="item.id"
 								:item="item"
 								is-search-result />
 						</template>
+
+						<!-- Search results: users -->
 						<template v-if="searchResultsUsers.length !== 0">
 							<NcAppNavigationCaption :title="t('spreed', 'Users')" />
 							<NcListItem v-for="item of searchResultsUsers"
 								:key="item.id"
 								:title="item.label"
 								@click="createAndJoinConversation(item)">
 								<template #icon>
-									<ConversationIcon :item="iconData(item)"
-										:disable-menu="true" />
-								</template>
-							</NcListItem>
-						</template>
-						<template v-if="!showStartConversationsOptions">
-							<NcAppNavigationCaption v-if="searchResultsUsers.length === 0"
-								:title="t('spreed', 'Users')" />
-							<Hint v-if="contactsLoading" :hint="t('spreed', 'Loading')" />
-							<Hint v-else :hint="t('spreed', 'No search results')" />
-						</template>
-					</template>
-					<template v-if="showStartConversationsOptions">
-						<template v-if="searchResultsGroups.length !== 0">
-							<NcAppNavigationCaption :title="t('spreed', 'Groups')" />
-							<NcListItem v-for="item of searchResultsGroups"
-								:key="item.id"
-								:title="item.label"
-								@click="createAndJoinConversation(item)">
-								<template #icon>
-									<ConversationIcon :item="iconData(item)"
-										:disable-menu="true" />
+									<ConversationIcon :item="iconData(item)" disable-menu />
 								</template>
 							</NcListItem>
 						</template>
 
-						<template v-if="searchResultsCircles.length !== 0">
-							<NcAppNavigationCaption :title="t('spreed', 'Circles')" />
-							<NcListItem v-for="item of searchResultsCircles"
-								:key="item.id"
-								:title="item.label"
-								@click="createAndJoinConversation(item)">
-								<template #icon>
-									<ConversationIcon :item="iconData(item)"
-										:disable-menu="true" />
-								</template>
-							</NcListItem>
+						<!-- Search results: new conversations -->
+						<template v-if="showStartConversationsOptions">
```

Comment:
```suggestion
						<template v-if="canStartConversations">
```

Method is redundant, it uses `isSearching && canStartConversations` inside of the same template